website/docusaurus.config.ts

```diff
@@ -466,6 +466,8 @@ export default async function createConfigAsync() {
                 label: `${getNextVersionName()} 🚧`,
               },
             },
+            tagsFilePath: 'tags.yml',
```

Comment:
feature is opt in so if we pass no options it should not be "executed" however this is the case for now